### PR TITLE
translated README.md to Simplified Chinese

### PR DESCRIPTION
**Pull request by mrmrmrfinch**

### CODE REVIEW

**Comment by mrmrmrfinch** [comment]
@jlooper Is there an official translation for the term "Azure Cloud Advocate?" I translated that into something that's close to "The advocates of Azure Cloud." I wonder if that's okay.

**Comment by jlooper** [comment]
> @jlooper Is there an official translation for the term "Azure Cloud Advocate?" I translated that into something that's close to "The advocates of Azure Cloud." I wonder if that's okay.

I don't know of one....thank you, this sounds good to me!

File: translations/README.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+> 如果希望进一步学习，我们推荐跟随 [Microsoft Learn](https://docs.microsoft.com/en-us/users/jenlooper-2911/collections/k7o7tg1gp306q4?WT.mc_id=academic-15963-cxa) 的模块和学习路径。
+
+**对于老师们**，我们对于如何使用这套教程[提供了一些建议](for-teachers.md)。
+
+---
+
+## 项目团队
+
+[![宣传视频](ml-for-beginners.png)](https://youtu.be/Tj1XWrDSYJU "宣传视频")
```
Comment (jlooper) [inline translations/README.zh-cn.md:46]: hi, can you edit this to be `../ml-for-beginners.png` so the path is right - thanks!

**Review by jlooper — CHANGES_REQUESTED** [review]

File: translations/README.zh-cn.md (hunk 1 above)
Comment (jlooper) [reply translations/README.zh-cn.md:46]: let's also make sure all the paths to lessons are correct

Comment (mrmrmrfinch) [reply translations/README.zh-cn.md:46]: Of course.

**Comment by mrmrmrfinch** [comment]
Sorry, I was working on the wrong file. It should be okay now!

**Comment by jlooper** [comment]
looks good, we will need to potentially change the lesson links after they are all translated - thanks!

**Pull request merged**